jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -499,16 +499,11 @@ private void checkImageInTargetRegistry() {
         executorService.submit(
             () -> {
               Verify.verify(results.builtImagesAndBaseImages.get().size() == 1);
```

Comment:
Since we've removed the `iterator().next()` code below and the `CheckImageStep` code can fully work without the assumption that there should be a single image, I think we can remove this verify line.

Reply:
Good Point